docs/source/system-requirements.rst

```diff
@@ -0,0 +1,59 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+System Requirements
+===================
+
+Hardware Requirements
+*********************
+
+.. tabs::
+
+   .. tab:: GPU
+
+      - 6th Gen Intel® Core™ processor or higher
+      - Intel® Iris® Plus Graphics
+      - Intel® Iris® Xe Graphics
+      - Intel® Iris® Xe Max Graphics
+      - Intel® Iris® Graphics
+      - Intel® Iris® Pro Graphics
+
+   .. tab:: CPU
+
+      - Intel Atom® Processors
+      - Intel® Core™ Processor Family
+      - Intel® Xeon® Processor Family
+      - Intel® Xeon® Scalable Performance Processor Family
+
```

Comment:
Let's add "Generic X86 processor"  - or might be other proper way to name non IA CPUs

Reply:
done